src/test/java/com/facebook/openwifirrm/optimizers/TestUtils.java

```diff
@@ -26,6 +28,8 @@ public class TestUtils {
 	/** The Gson instance. */
 	private static final Gson gson = new Gson();
 
+	public static final Instant DEFAULT_START_TIME = Instant.parse("2022-01-01T00:00:00Z");
```

Comment:
this needs a comment and the name is also not descriptive enough (ex. `DEFAULT_WIFISCAN_START_TIME`)